frontend/src/_components/Pagination.jsx

```diff
@@ -52,7 +52,13 @@ export const Pagination = function Pagination({ currentPage, count, pageChanged,
   }
 
   return (
-    <div className="card-footer d-flex align-items-center px-1">
+    <div
+      className={
```

Comment:
```
className=`card-footer d-flex align-items-center px-1 ${darkMode ? 'bg-transparent' : ''}`
```

Something like this might be easier to read.

Reply:
Made the required changes